packages/files-ui/src/Contexts/ThresholdKeyContext.tsx

```diff
@@ -396,6 +399,31 @@ const ThresholdKeyProvider = ({ children, network = "mainnet", enableLogging = f
     }
   }, [userInfo, address])
 
+  const createJWT = useCallback((bucketId: string, nonceId: string, permission: NonceResponsePermission) => {
+
+    if(!privateKey) {
+      console.log("no private key found")
```

Comment:
```suggestion
```

Reply:
I'd keep this one, or set it to a console.error
```suggestion
      console.error("no private key found")
```